packages/@react-spectrum/s2/intl/en-US.json

```diff
@@ -25,6 +25,7 @@
   "table.sortAscending": "Sort Ascending",
   "table.sortDescending": "Sort Descending",
   "table.resizeColumn": "Resize column",
+  "table.standardColumnMenu": "Default column actions",
```

Comment:
Need a better menu section name

Reply:
hmm does the section have to have it's own label? is that required by aria? Maybe I'd just say "Column actions" - default is kind of an implementation detail no?